docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+[source, subs="+quotes"]
+----
+export OFFLINE_TOKEN=<offline_token>
+----
+====
+
+.Procedure
+
+. Open the `main.tf` file in your IDE for editing.
+. Enter the example rhoas_kafka resource. This example uses the `"my-instance"`  identifier and creates a Kafka instance called `my-instance`. The only required fields are the `name`, `billing_model` and `plan` fields. The following values are set by default when you run `terraform apply`.
++
+.. `cloud provider`
+.. `region`
++
+All other information for the instance is provided by the Kafka APIs. The generated bootstrap server displays in the terminal as an output.
++
+.Example `rhoas_kafka` resource
+[source]
+----
+resource "rhoas_kafka" "my-instance" {
+  name = "my-instance"
+  plan = "developer.x1"
+  billing_model = "standard"
+}
+  output "bootstrap_server_my-instance" {
+    value = rhoas_kafka.my-instance.bootstrap_server_host
+}
+
+----
+. After creating your Kafka instance, you create a service account. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. In the `main.tf` file, enter the example `rhoas_service_account` resource to create a service account. This example uses the `"my-service-account"` identifier and creates a service account called my-service-account. The generated client ID displays in the terminal as an output.
++
+
+.Example `rhoas_service_account` resource
+[source]
+----
+resource "rhoas_service_account" "my-service-account" {
+  name        = "my-service-account"
+  description = "<description of service account>"
+}
+
+output "client_id" {
+  value = rhoas_service_account.my-service-account.client_id
+}
+
+output "client_secret" {
+  value     = rhoas_service_account.my-service-account.client_secret
+  sensitive = true
+}
+----
++
+The values for the client ID and client secret will be set owhen you run `terraform apply`.
+. Enter the example `rhoas_topic` resource to create a Kafka topic with default values. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. As you have already created the Kafka instance, Terraform can check dependencies for this new topic resource and knows the Kafka ID when you run this example resource.
++
+.Example `rhoas_topic` resource with default values
+[source]
+----
+resource "rhoas_topic" "topic" {
+		name = "my-topic"
+		partitions = 1
+		kafka_id = rhoas_kafka.instance.id
+	}
+
+----
++
+. Enter the `rhoas_acl` resource to create an ACL binding. This example uses the `"acl"` identifier. The following list is a description of each field in an ACL binding resource.
+
+* `resource_type`: The type of resource you want access to. This example uses *“TOPIC”*.
```

Comment:
Same as https://github.com/redhat-developer/app-services-guides/pull/601/files#r1034563631, I'd format this is a definition list.

Also:
- Fix the inconsistent capitalization in the descriptions.
- This example uses `"TOPIC"` (backticks, not asterisks)